editor/modes/visual-editor/block-drop-zone.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+import { reduce, get, find } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { DropZone } from '@wordpress/components';
+import { getBlockTypes } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { insertBlocks } from '../../actions';
+
+function BlockDropZone( { index, ...props } ) {
+	const dropFiles = ( files, position ) => {
+		const transformation = reduce( getBlockTypes(), ( ret, blockType ) => {
```

Comment:
Should this be `find` instead of `reduce`? Why continue iterating the list if we only want first result?

```js
const transformation = find( getBlockTypes(), ( ret, blockType ) => {
	return find( get( blockType, 'transforms.from', [] ), ( transform ) => (
		transform.type === 'files' && transform.isMatch( files )
	) );
} );
```